plugins/microsoft_intune/icon_microsoft_intune/util/api.py

```diff
@@ -214,3 +304,15 @@ def create_necessary_headers(access_token):
             "Authorization": f"Bearer {access_token}",
             "Content-Type": "application/json",
         }
+
+    def _filter_policy_id(self, managed_app_policies, policy_name):
+        for item in managed_app_policies["value"]:
```

Comment:
```suggestion
        for item in managed_app_policies.get("value"):
```
Prevent `KeyError` if `"value"` not in `managed_app_policies`